docs/Classes/NetworkConnection.md

```diff
@@ -14,8 +14,6 @@ NetworkConnection is a high-level API class that encapsulates a network connecti
     The last time that a message was received on this connection.
 -   **address**  
     The IP address of the end-point that this connection is connected to.
--   **playerControllers**  
```

Comment:
we removed playerControllers,  but we added playerController  (singular),  so perhaps you can change the comment to describe playerController?

Reply:
I've committed a new change... Hopefully more appropriate.